lib/inferno/dsl/runnable.rb

```diff
@@ -355,6 +356,22 @@ def route(method, path, handler)
       def test_count
         @test_count ||= children&.reduce(0) { |sum, child| sum + child.test_count } || 0
       end
+
+      def contained_required_inputs(prior_outputs = [])
```

Comment:
This name confuses me. I think this method should be `required_inputs`. I'd also remove `get_` from `get_missing_inputs`, as I don't think it really adds anything.